Eatery/Onboarding/Controllers/OnboardingPageViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  OnboardingPageViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/12/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+import CHIPageControl
+
+class OnboardingPageViewController: UIPageViewController {
+
+    var pages = [OnboardingViewController]()
+    let pageControl = CHIPageControlJaloro()
+
+    let models = [
+        OnboardingModel(title: "Menus", subtitle: "See what’s being served at any campus eatery.", image: UIImage(named: "menuIcon")!),
```

Comment:
Instead of using named UIImages, why not use Image Literals? That way you avoid force unwraps.

Reply:
I disagree: image literals are hard to manage in the source code editor

Reply:
This whole `OnboardingModel` thing is a bit too much IMO. Remove `OnboardingModel` and maybe pass in the image, title, subtitle into the `OnboardingViewController` directly?